Example/CryptoOnramp Example/CryptoOnramp Example/IdentityVerificationView.swift

```diff
@@ -31,7 +31,7 @@ struct IdentityVerificationView: View {
                     .font(.largeTitle)
                     .padding()
                     .background {
-                        Color.secondary.opacity(0.2)
+                        Color(.systemGroupedBackground)
```

Comment:
This is to match the color I’m using for a similar view in `WalletSelectionView` (the background of the SF Symbol near the top).